### ISSUE TITLE
separate web frontend?

### ISSUE THREAD

**Issue by dweinstein** [alibaba/anyproxy#26]
what do you guys think about a separate module for the web frontend and the backend

**Comment by ottomao** [comment]
i dont think it neccessary since our web frontend is still very poor. Maybe we will pack the frontend as an individual module after we make it strong enough.
